verifier/src/x509/algorithm.rs

```diff
@@ -0,0 +1,444 @@
+// Copyright (c) 2023 The MobileCoin Foundation
+
+//! Algorithm data types used in x509 certificate logic
```

Comment:
Up for suggestions on this naming. "algorithm" feels a little odd of a name, this is the signature and keys. Maybe each type should go to a separate module?